### DISCUSSION TITLE
[Projects Beta] Keyboard shortcuts

### DISCUSSION THREAD

**Discussion by josh-lester** [community/community#5890] [Projects and Issues]
I'm not certain if this is already implemented and I can not figure out the command or not.

Currently I can select an issue with the mouse and move the issue up or down the list.

It would be nice as a user to be able to select an issue in any view / board and rearrange it on the list. Ex. I select the issue, using on the keyboard be able to move the issue up the list or down the list.

**Comment by PabloLION** [comment]
I think all keyboard shortcuts are written here. (Type question mark(Shift+/), with no focused element on this page, if you have a US keyboard, and you'll see the pop-up.)
I wanted to make a Feature Request days ago (see [my wiki](https://github.com/Pablion/Pablion/wiki#feature-request)) and you are 8 min ealier than me.

**Comment by pyrello** [comment]
Related: https://github.com/github/feedback/discussions/6610
